src/main/java/org/osiam/metrics/controller/MetricsController.java

```diff
@@ -64,7 +67,7 @@
     private ObjectMapper createMetricsJSONMapper() {
         final TimeUnit rateUnit = parseTimeUnit(RATE_UNIT, TimeUnit.SECONDS);
         final TimeUnit durationUnit = parseTimeUnit(DURATION_UNIT, TimeUnit.SECONDS);
-        return new ObjectMapper().registerModule(new MetricsModule(rateUnit, durationUnit, false));
+        return mapper.registerModule(new MetricsModule(rateUnit, durationUnit, false));
```

Comment:
This will be called every time someone request the metrics and will fail if it's called the second time. You cannot add the same module twice.